website/tests/test_main.py

```
@@ -40,34 +40,57 @@ class MySeleniumTests(LiveServerTestCase):
 
+    def setUp(self):
+        """Single setUp method combining all setup logic"""
+        super().setUp()
+
+        # Clean up existing test users first
+        User.objects.filter(username__startswith="testuser_").delete()
+
+        # Create test user with unique username using timestamp
+        timestamp = int(time.time())
+        self.user = User.objects.create_user(
+            username=f"testuser_{timestamp}", email=f"testuser_{timestamp}@example.com", password="secret"
```

Ruff (0.14.5)

Possible hardcoded password assigned to argument: "password"

(S106)